src/components/Anchor/index.js

```diff
@@ -21,13 +21,14 @@ const prefixByType = {
 const Anchor = ({
   url,
   children,
+  className = '',
   isSeparateTab = false,
   type = 'inline',
 }) => (
   <a
     href={url}
     target={isSeparateTab ? '_blank' : '_self'}
-    className={classNameByType[type]}
+    className={`${classNameByType[type]} ${className}`}
```

Comment:
I've developed this after a year and a half working on the Domo app; I used to use `className` and `{...this.props}` everywhere when I first started with React, but over time it became very difficult to maintain and changing things broke other things. Having explicit props and wrapper side effects scales much easier in my experience.